webserver/webserver/src/main/java/io/helidon/webserver/ConfiguredTlsManager.java

```diff
@@ -0,0 +1,278 @@
+                                   String type) {
+        this.name = Objects.requireNonNull(name);
+        this.type = Objects.requireNonNull(type);
+    }
+
+    /**
+     * The manager name.
+     *
+     * @return manager name
+     */
+    // @Override // NamedService
+    public String name() {
+        return name;
+    }
+
+    /**
+     * The manager type.
+     *
+     * @return manager type
+     */
+    // @Override // NamedService
+    public String type() {
+        return type;
+    }
+
+    @Override // TlsManager
+    public SSLContext sslContext() {
+        return sslContext;
+    }
+
+    @Override // TlsManager
+    public void subscribe(Consumer<SSLContext> sslContextConsumer) {
+        subscribers.add(Objects.requireNonNull(sslContextConsumer));
+    }
+
+    @Override // TlsManager
+    public void init(WebServerTls tlsConfig) {
+        SSLContext explicitSslContext = tlsConfig.explicitSslContext().orElse(null);
+        if (explicitSslContext != null) {
+            this.sslContext = explicitSslContext;
+            return;
+        }
+
+        if (tlsConfig.privateKeyConfig() == null) {
+            throw new IllegalStateException("Private key must be configured when SSL is enabled.");
+        }
+
+        try {
+            KeyManagerFactory kmf = buildKmf(tlsConfig.privateKeyConfig());
+            TrustManagerFactory tmf = buildTmf(tlsConfig);
+
+            initSslContext(tlsConfig,
+                           kmf.getKeyManagers(),
+                           tmf.getTrustManagers());
+        } catch (IOException | GeneralSecurityException e) {
+            throw new IllegalStateException("Failed to build server SSL Context!", e);
+        }
+    }
+
+    /**
+     * Initialize and set the SSL context given the provided configuration.
+     *
+     * @param tlsConfig     the tls config
+     * @param keyManagers   the key managers
+     * @param trustManagers the trust managers
+     * @deprecated this method will removed in a future release.
+     */
+    @Deprecated
+    protected void initSslContext(WebServerTls tlsConfig,
+                                  KeyManager[] keyManagers,
+                                  TrustManager[] trustManagers) {
+        try {
+            // Initialize the SSLContext to work with our key managers.
+            SSLContext sslContext = SSLContext.getInstance(tlsConfig.protocol());
+            sslContext.init(keyManagers, trustManagers, secureRandom(tlsConfig));
+
+            configureAndSet(tlsConfig, sslContext);
+        } catch (GeneralSecurityException e) {
+            throw new IllegalArgumentException("Failed to create SSLContext", e);
+        }
+    }
+
+    /**
+     * Called when there is a new {@link SSLContext}.
+     *
+     * @param tlsConfig     the tls config
+     * @param keyManagers   the key managers
+     * @param trustManagers the trust managers
+     * @deprecated this method will removed in a future release.
```

Comment:
I think you are missing "be" -> will be removed

Reply:
It is also in some other places in this class. Please check that.

Reply:
pushed